cyclus_input/predicting_the_past.xml

```diff
@@ -0,0 +1,183 @@
+    <spec><lib>agents</lib>   <name>NullRegion</name> </spec>
+  </archetypes>
+  <xi:include href="./buildtimes/buildtimes.xml#xpointer(/simulation/facilities/child::*)"/>
+  <facility>
+    <name>Sink1</name>
+    <config>
+      <Sink>
+        <in_commods>
+          <val>casked_spent_uox</val>
+        </in_commods>
+      </Sink>
+    </config>
+  </facility>
+  <facility>
+    <name>Mine1</name>
+    <config>
+      <Source>
+        <outcommod>nat_u</outcommod>
+        <outrecipe>nat_u</outrecipe>
+        <throughput>1384555031.396084452</throughput>
+      </Source>
+    </config>
+  </facility>
+  <facility>
+    <name>Mill1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>nat_u</val>
+        </in_commods>
+        <out_commods>
+          <val>u3o8</val>
+        </out_commods>
+        <residence_time>1</residence_time>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>Conversion1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>u3o8</val>
+        </in_commods>
+        <out_commods>
+          <val>uf6</val>
+        </out_commods>
+        <residence_time>1</residence_time>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>Enrichment1</name>
+    <config>
+      <Enrichment>
+        <feed_commod>uf6</feed_commod>
+        <feed_recipe>uf6</feed_recipe>
+        <product_commod>enr_u</product_commod>
+        <tails_commod>tails</tails_commod>
+        <max_enrich>0.045</max_enrich>
+      </Enrichment>
+    </config>
+  </facility>
+  <facility>
+    <name>FuelFab1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>enr_u</val>
+        </in_commods>
+        <out_commods>
+          <val>fresh_uox</val>
+        </out_commods>
+        <residence_time>1</residence_time>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>WetStorage1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>spent_uox</val>
+        </in_commods>
+        <out_commods>
+          <val>cool_spent_uox</val>
+        </out_commods>
+        <residence_time>72</residence_time>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>DryStorage1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>cool_spent_uox</val>
+        </in_commods>
+        <out_commods>
+          <val>casked_spent_uox</val>
+        </out_commods>
+      </Storage>
+    </config>
+  </facility>
+  <region>
+    <name>US</name>
+    <config>
+      <NullRegion/>
+    </config>
+    <institution>
+      <name>FuelCycle</name>
+      <initialfacilitylist>
+        <entry>
+          <prototype>Mine1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>Mill1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>Conversion1</prototype>
```

Comment:
You're right. This is unnecessary.
I initially added them, thinking I might have to add more facilities of certain types for a more complex simulation.